### PR TITLE
Add tests for Renamer when using globals with display names

### PR DESCRIPTION
**Pull request by lesaltzm**
From a comment on #503

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs [hunk 1]
```diff
@@ -47,11 +47,10 @@ public IEnumerable<string> GetAllFunctionNames()
         /// Create a resolver for use in binding. This is called from <see cref="Check(string, RecordType, ParserOptions)"/>.
         /// Base classes can override this is there are additional symbols not in the config.
         /// </summary>
-        /// <param name="alternateConfig">An alternate config that can be provided. Should default to engine's config if null.</param>        
         /// <returns></returns>
-        private protected virtual INameResolver CreateResolver(PowerFxConfig alternateConfig = null)
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs:52]: >PowerFxConfig alternateConfig = null [](http://example.com/codeflow?start=63&length=36)

Great to remove this!
FYI, technically an internal breaking change - will require a trivia update in PAClient.

File: src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs [hunk 2]
```diff
@@ -198,16 +198,6 @@ internal bool TryGetSymbol(DName name, out IExternalEntity symbol, out DName dis
...
 
             return _environmentSymbols.TryGetValue(lookupName, out symbol);
-        }
-
-        /// <summary>
-        /// Some scenarios require that lookups be done with logical names only.
-        /// This returns the same PowerFxConfig with the display name provider disabled.
-        /// </summary>
-        /// <returns></returns>
-        internal PowerFxConfig WithoutDisplayNames()
```
Comment (MikeStall) [inline src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs:208]: >WithoutDisplayNames [](http://example.com/codeflow?start=31&length=19)

yes!!!

**Review by MikeStall — APPROVED** [review]
:shipit:

File: src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs (hunk 1 above)
Comment (lesaltzm) [reply src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs:52]: Yeah, will pick up with #520

**Pull request merged**